awx/main/models/credential/__init__.py

```diff
@@ -1173,6 +1178,38 @@ def create(self):
 )
 
 
+ManagedCredentialType(
+    namespace='galaxy_api_token',
+    kind='galaxy',
+    name=ugettext_noop('Ansible Galaxy/Automation Hub API Token'),
```

Comment:
These highly descriptive names are kind of messy from the user's perspective using the name to reference the object, from the CLI or the collection like

```yaml
tower_credential:
  credential_type: "Ansible Galaxy/Automation Hub API Token"
```

Reply:
but no change here makes a lot of sense. We might want to add an issue to reference credential types by `namespace` in the collection modules.